internal/clusterinfo/clusterinfo.go

```diff
@@ -76,14 +79,36 @@ func Discover(ctx context.Context, kubeClient kubernetes.Interface) (Info, error
 }
 
 // IsManagedKyma returns true if the k8s go-client is configured to access a managed kyma runtime
-func IsManagedKyma(restConfig *rest.Config) bool {
-	//Legacy, may be removed in the future
+func IsManagedKyma(ctx context.Context, restConfig *rest.Config, kubeClient kubernetes.Interface) (bool, error) {
+	//1) Verfiy the domain
 	if strings.HasSuffix(restConfig.Host, legacyEnvAPIServerSuffix) {
-		return true
+		//Legacy, may be removed in the future
```

Comment:
This comment also provides no informational value since the const ist already explaining legacy context. Can be removed?

Reply:
:+1: